packages/clerk-js/src/ui/components/OrganizationProfile/VerifiedDomainPage.tsx

```diff
@@ -1,29 +1,42 @@
 import type { OrganizationEnrollmentMode } from '@clerk/types';
 
 import { useCoreOrganization, useEnvironment } from '../../contexts';
-import { Col, descriptors, Flex, localizationKeys, Spinner } from '../../customizables';
+import { Col, Flex, localizationKeys, Spinner } from '../../customizables';
 import {
-  BlockWithAction,
   ContentPage,
   Form,
   FormButtons,
   Header,
+  Tab,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  TabsList,
   useCardState,
   withCardStateProvider,
 } from '../../elements';
-import { useFetch } from '../../hooks';
+import { useFetch, useNavigateToFlowStart } from '../../hooks';
 import { useRouter } from '../../router';
 import { handleError, useFormControl } from '../../utils';
-import { EnrollmentBadge } from './EnrollmentBadge';
+import { LinkButtonWithDescription } from '../UserProfile/LinkButtonWithDescription';
 import { OrganizationProfileBreadcrumbs } from './OrganizationProfileNavbar';
 
 export const VerifiedDomainPage = withCardStateProvider(() => {
   const card = useCardState();
   const { organizationSettings } = useEnvironment();
   const { organization } = useCoreOrganization();
-  const { params, navigate } = useRouter();
+  const { domains } = useCoreOrganization({
+    domains: {
+      infinite: true,
+    },
+  });
+  const { navigateToFlowStart } = useNavigateToFlowStart();
+  const { params, navigate, queryParams } = useRouter();
+  const mode = (queryParams.mode ?? 'edit') as 'select' | 'edit';
```

Comment:
```suggestion
  const mode = (queryParams.mode ||'edit') as 'select' | 'edit';
```

If we don't care about `null`, `undefined` or `0` specifically, let's prefer a plain `||` otherwise this reads like the falsy values have significant meaning

Reply:
[Here's how](https://github.com/search?q=repo%3Aclerkinc%2Fjavascript%20return%20navigate(%60..%2Freset-password-success%3F%24%7BqueryParams.toString()%20%7D%60)%3B&type=code) we did it for the reset password flow. This was implemented by me as well. 

Are we considering this a bad pattern ?